### PR TITLE
clientID added to resume

### PR DESCRIPTION
**Pull request by leonjoyce**
According to the following discussion, added client ID to bootstrap.resume
https://github.com/zendesk/maxwell/issues/1041

### CODE REVIEW

File: src/main/java/com/zendesk/maxwell/bootstrap/SynchronousBootstrapper.java [hunk 1]
```diff
@@ -150,7 +152,8 @@ public void resume(AbstractProducer producer, Replicator replicator) throws Exce
 			// This update resets all rows of incomplete bootstraps to their original state.
 			// These updates are treated as fresh bootstrap requests and trigger a restart
 			// of the bootstrap process from the beginning.
-			String sql = "update `bootstrap` set started_at = NULL where is_complete = 0 and started_at is not NULL";
+			String clientid = ((BinlogConnectorReplicator)replicator).getClientID();
```
Comment (osheroff) [inline src/main/java/com/zendesk/maxwell/bootstrap/SynchronousBootstrapper.java]: better to extract `clientID` from `context`

File: src/main/java/com/zendesk/maxwell/bootstrap/SynchronousBootstrapper.java [hunk 2]
```diff
@@ -150,7 +152,8 @@ public void resume(AbstractProducer producer, Replicator replicator) throws Exce
...
+			String sql = "update `bootstrap` set started_at = NULL where is_complete = 0 and started_at is not NULL and client_id = '" + clientid + "'";
```
Comment (osheroff) [inline src/main/java/com/zendesk/maxwell/bootstrap/SynchronousBootstrapper.java]: this needs to use regular sql quoting via `client_id = ?` and `PreparedStatement#setString()`

File: src/main/java/com/zendesk/maxwell/bootstrap/SynchronousBootstrapper.java (hunk 1 above)
Comment (leonjoyce) [reply src/main/java/com/zendesk/maxwell/bootstrap/SynchronousBootstrapper.java]: Agree

File: src/main/java/com/zendesk/maxwell/bootstrap/SynchronousBootstrapper.java (hunk 2 above)
Comment (leonjoyce) [reply src/main/java/com/zendesk/maxwell/bootstrap/SynchronousBootstrapper.java]: I fix the code according to review comments. But not sure why the build fails. Can You help?

**Comment by osheroff** [comment]
this change got merged in another PR

**Pull request closed without merging**